Home-exercises/src/main/java/io/codelex/fiveTasks/FirstTask/Main.java

```diff
@@ -0,0 +1,49 @@
+package io.codelex.fiveTasks.FirstTask;
+
+import java.math.BigDecimal;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Please input card number: ");
+        int number = sc.nextInt();
+        sc.nextLine();
+        String fullName = sc.nextLine();
+        System.out.print("Please input full name: ");
+        boolean isVal = false;
+        String CCVcode = "";
+        System.out.print("Please input CCV code: ");
+        while (!isVal) {
+            CCVcode = sc.nextLine();
+            isVal = validateCCV(CCVcode);
+        }
+
+
+        BigDecimal balance = sc.nextBigDecimal();
+        System.out.println("Please input your balance: ");
+        CreditCard creditCard = new CreditCard(number, fullName, CCVcode, balance);
+        creditCard.withdraw();
+
+    }
+
+    public static boolean validateCCV(String CCVcode) {
```

Comment:
Arī šis faktiski ir lieks, jo uzdevumā (kā arī citās programmu specifikācijās) minētais "CCV code (three digit number but it can start with 0, 045 for example)" - ar to tiek aprakstīti dati, lai izvēlētos konkrētu datu tipu. Bet izdomāji veikt validāciju, kas nebija nepieciešams, jo lietotāja ievade vispār nebija paredzēta.